Phases/Assessments/Phase_2.md

```diff
@@ -0,0 +1,38 @@
+# Phase 2 Assessment
+
+The phase 2 assessment will test your ability to write more complex JavaScript programs for the Node.js platform, to create simple HTTP applications, to write HTML & CSS to give structure and style to a web page, and to design and interact with more complex relational databases.
```

Comment:
`and to design and interact with more complex relational databases.` -> `and delve deeper into SQL schema design and queries.`